pkg/disk/partition_table_test.go

```diff
@@ -2130,3 +2282,23 @@ func TestNewCustomPartitionTableErrors(t *testing.T) {
 		})
 	}
 }
+
+func TestPartitionTableFeatures(t *testing.T) {
+	type testCase struct {
+		partitionType    string
+		expectedFeatures disk.PartitionTableFeatures
+	}
+	testCases := []testCase{
+		{"plain", disk.PartitionTableFeatures{XFS: true, FAT: true}},
+		{"plain-swap", disk.PartitionTableFeatures{XFS: true, FAT: true}},
+		{"luks", disk.PartitionTableFeatures{XFS: true, FAT: true, LUKS: true}},
+		{"luks+lvm", disk.PartitionTableFeatures{XFS: true, FAT: true, LUKS: true, LVM: true}},
+		{"btrfs", disk.PartitionTableFeatures{XFS: true, FAT: true, Btrfs: true}},
```

Comment:
I don't know if it's surprising or not.  Depends on the understanding and meaning of "features" here, so I think it would be more appropriate to describe it in the `features()` function itself rather than the test.

Reply:
Will add to the features follow-up.